From cf953d2bfa7df9aa67459b333db4d4d8a9e72fd6 Mon Sep 17 00:00:00 2001
From: Thomas Monjalon <thomas.monjalon@6wind.com>
Date: Fri, 27 Jun 2014 11:21:11 +0200
Subject: [PATCH] app/test: fix build switches to enable cmdline tests

Upstream-Status: backport
Imported patch from: http://dpdk.org/browse/dpdk/log/

There were 2 typos since these commits (in 1.6.0 releases):
	21a7f4e264 fix build without librte_cmdline
	cac6d08c8b replace --use-device option by --pci-whitelist and --vdev
In makefiles, the build options are prefixed with CONFIG_RTE_
but in .c file, it is only RTE_.

These typos were disabling cmdline unit tests and test of "--vdev eth_ring" option.

Signed-off-by: Thomas Monjalon <thomas.monjalon@6wind.com>
Acked-by: Pablo de Lara <pablo.de.lara.guarch@intel.com>
Signed-off-by: Chan Wei Sern <wei.sern.chan@intel.com>
---
 app/test/test_cmdline.c   | 2 +-
 app/test/test_eal_flags.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/test/test_cmdline.c b/app/test/test_cmdline.c
index 77475c4..10a3f77 100644
--- a/app/test/test_cmdline.c
+++ b/app/test/test_cmdline.c
@@ -39,7 +39,7 @@
 int
 test_cmdline(void)
 {
-#ifdef CONFIG_RTE_LIBRTE_CMDLINE
+#ifdef RTE_LIBRTE_CMDLINE
 	printf("Testind parsing ethernet addresses...\n");
 	if (test_parse_etheraddr_valid() < 0)
 		return -1;
diff --git a/app/test/test_eal_flags.c b/app/test/test_eal_flags.c
index a862654..1b80b80 100644
--- a/app/test/test_eal_flags.c
+++ b/app/test/test_eal_flags.c
@@ -317,7 +317,7 @@ test_whitelist_flag(void)
 	const char *wlval3[] = {prgname, prefix, mp_flag, "-n", "1", "-c", "1",
 			pci_whitelist, "09:0B.3,type=test",
 			pci_whitelist, "08:00.1,type=normal",
-#ifdef CONFIG_RTE_LIBRTE_PMD_RING
+#ifdef RTE_LIBRTE_PMD_RING
 			vdev, "eth_ring,arg=test",
 #endif
 	};
-- 
1.9.1

